app/frontend/src/features/BugReport.tsx

```diff
@@ -103,54 +127,43 @@ export default function BugReport() {
         aria-labelledby="bug-reporter"
         open={isOpen}
         onClose={handleClose}
+        // won't close when clicking outside
+        disableBackdropClick
+        // won't clsoe when pressing esc
+        disableEscapeKeyDown
```

Comment:
Why are we disabling this? That seems to go against the default expected behaviour of a modal...

Reply:
People have been complaining about accidentally clicking outside the modal and then losing their hard work in writing a bug report. That I think is fair enough.

Happy to remove the escape disable though.

What do you think?

Reply:
I see... maybe doing this as a modal isn't the best way?

Reply:
I think it's quite handy as a modal: people don't have to navigate back and forth when reporting bugs?

Reply:
Yeah ok, let's re-enable escape but disable background click. Just looked around and that's not actually an expected behaviour in terms of accessibility, so it must be an additional "nice to have" pattern people/Material UI implement